### PR TITLE
Ch10590

### PR DESCRIPTION
**Pull request by adamsdarlingtower**
Adds casts to functions used as cleaners in transforms

### CODE REVIEW

File: plaidcloud/utilities/sqlalchemy_functions.py [hunk 1]
```diff
@@ -359,53 +397,144 @@ def compile_safe_to_date(element, compiler, **kw):
...
 class sql_safe_divide(GenericFunction):
     name = 'safe_divide'
 
-
 @compiles(sql_safe_divide)
 def compile_safe_divide(element, compiler, **kw):
     """Divides numerator by denominator, returning NULL if the denominator is 0.
     """
     numerator, denominator, divide_by_zero_value = list(element.clauses)
+    numerator = func.cast(number, sqlalchemy.Numeric)
```
Comment (rad-pat) [inline plaidcloud/utilities/sqlalchemy_functions.py:536]: Where does number com from?

File: plaidcloud/utilities/sqlalchemy_functions.py [hunk 2]
```diff
@@ -359,53 +397,144 @@ def compile_safe_to_date(element, compiler, **kw):
...
+    denominator = func.cast(digits, sqlalchemy.Integer)
```
Comment (rad-pat) [inline plaidcloud/utilities/sqlalchemy_functions.py:537]: Where does digits come from?

**Review by rad-pat — CHANGES_REQUESTED** [review]
I think the only problems are the undefined variables?

File: plaidcloud/utilities/sqlalchemy_functions.py (hunk 1 above)
Comment (adamsdarlingtower) [reply plaidcloud/utilities/sqlalchemy_functions.py:536]: ah, good catch, thank you

**Review by rad-pat — APPROVED** [review]

**Pull request merged**